lib/core/DocsLayout.js

```diff
@@ -110,6 +115,12 @@ class DocsLayout extends React.Component {
                 </a>
               )}
             </div>
+            {updateTime !== null ? (
```

Comment:
Try [Inline If with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) which is more declarative

Example
```js
{updateTime && <div>{updateTime}</div>}
```

Reply:
@endiliey Thanks for approving. Yes you are right, adding tests would be a great thing to do. But I had a few doubts regarding that. Firstly, when testing should I compare the timestamp returned to some hard-coded timestamp or just that it's a string. Because the hardcided timestamp would change of someone changes a doc. Secondly, if git is not installed then these tests won't work right?

Reply:
You should use a test docs and put it in __fixtures__. The test could be a snapshot testing. It wont change because we wont edit that test docs

Then the second case is to test if there is no git timestamp, the function should return null.